adb_graphics/default_specs.yml

```diff
@@ -1345,6 +1352,7 @@ trc1:
     clevs: [1, 2, 4, 6, 8, 12, 16, 20, 25, 30, 40, 60, 100, 200]
     colors: smoke_colors
     ncl_name: MASSDEN_P0_L103_{grid}
+    plot_airports: False
```

Comment:
Do we want to turn these off for all projections?

Reply:
Ravan has asserted that he doesn't want any airports plotted on any of the smoke-related maps, so I think this is ok.